client/src/main/java/com/alibaba/nacos/client/security/SecurityProxy.java

```diff
@@ -125,13 +127,13 @@ public boolean login(List<String> servers) {
      * @param server server address
      * @return true if login successfully
      */
-    public boolean login(String server) {
+    public boolean login(String server) throws UnsupportedEncodingException {
         
         if (StringUtils.isNotBlank(username)) {
             Map<String, String> params = new HashMap<String, String>(2);
             Map<String, String> bodyMap = new HashMap<String, String>(2);
             params.put("username", username);
-            bodyMap.put("password", password);
+            bodyMap.put("password", URLEncoder.encode(password, "utf-8"));
```

Comment:
This change has been revert, please check in `develop` branch.

Please do not add back.